### PR TITLE
Merge Develop into Master

### PR DESCRIPTION
**Pull request by ghost**
Project is complete and ready for review. Please provide feedback on any changes that need to be made.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+</head>
+
+<body>
+  <div class="container">
+    <div class="nav">
+      <div class="nav-text">
+        <p>Already have an Intuit account?</p>
+        <a href="#">I forgot my User ID or Password</a>
+      </div>
+      <div class="nav-button">
```
Comment (Torres-ssf) [inline index.html:18]: This div is a good candidate to be a main tag instead. Div tags have no semantic meaning, it's a much better practice to use the new HTML5's tag with semantic value. Check out this article to learn more about the importance of semantic tag https://www.lifewire.com/why-use-semantic-html-3468271.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          <a href="#">Learn more</a>
+        </div>
+        <form action="#">
+          <p>
+            <label for="email">Email address</label>
+            <input type="email" id="email" name="email">
+          </p>
+          <p>
+            <label for="phone">Phone(recommended)</label>
+            <input type="number" id="phone" name="phone">
```
Comment (Torres-ssf) [inline index.html]: There is a dedicated value for input's attribute type when you are using it for phone numbers; tel. Switch number for tel. You can check all values for the attribute type here https://www.w3schools.com/html/html_form_input_types.asp.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
Hi team. You did a great job on this project, it looks really nice 👏👏. It is almost ready to submit it. It only needs 2 small changes. Make these changes and submit to code review again. Happy coding

**Comment by Elukoye** [comment]
updated changes, please review.

**Comment by Torres-ssf** [comment]
I will give this project a 👍. Good work. You can merge to master. Don't forget to submit the project completion form.

**Comment by Elukoye** [comment]
Thank you Sergio Torres

**Pull request merged**